src/rpdk/core/invoke.py

```diff
@@ -110,6 +110,7 @@ def invoke(args):
             request["desiredResourceState"],
             request["previousResourceState"],
             request.get("typeConfiguration"),
+            stackId=request.get("stackId")
```

Comment:
Stack id is not always present, as this is purely CloudFormation concept. Resource Handlers are also used by CloudControl, which does not provide value for this field. I would prefer to left it out of `cfn invoke` payload.